include/proxy-wasm/word.h

```diff
@@ -24,11 +24,11 @@ namespace proxy_wasm {
 // Use byteswap functions only when compiling for big-endian platforms.
 #if defined(__BYTE_ORDER__) && defined(__ORDER_BIG_ENDIAN__) &&                                    \
     __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
-#define htowasm(x) __builtin_bswap32(x)
-#define wasmtoh(x) __builtin_bswap32(x)
+#define htowasm(x, is_wasm_byte_order) ((is_wasm_byte_order) ? __builtin_bswap32(x) : (x))
```

Comment:
Here, `is_wasm_byte_order` reflects whether the VM uses Wasm byte order. But if this section of code is read without the larger context of this PR (or looking at calling code), `is_wasm_byte_order` looks like it's referring to `x`, which then makes the macro appear inverted (even though it isn't).

This potential misreading could be avoided by naming the param `vm_is_wasm_byte_order` or `vm_uses_wasm_byte_order`, to make the subject of the phrase clear.

Reply:
Renamed to `vm_uses_wasm_byte_order`